src/main/java/com/coniverse/dangjang/domain/notification/service/NotificationService.java

```diff
@@ -97,6 +104,35 @@ public void saveFcmToken(String fcmToken, String oauthId) {
 	public void deleteFcmToken(String fcmToken) {
 		userFcmTokenRepository.findById(fcmToken).orElseThrow(InvalidFcmTokenException::new);
```

Comment:
P3) 로그아웃할 때 fcm 토큰이 없으면 예외를 던지지 않고
```suggestion
		try {
		    userFcmTokenRepository.deleteById(fcmToken);
		} catch (키가 없을 때 발생하는 예외 e) {
		    return; // 또는 기타 로직
		}
```
다른 로직으로 처리해야 하지 않나 싶어요
토큰이 존재하지 않아서 로그아웃되지 않는건 알맞지 않은 것 같아용